CoEpi/domain/logic/CenMatcher.swift

```diff
@@ -19,26 +20,31 @@ class CenMatcherImpl: CenMatcher {
 
     // Copied from Android implementation
     private func match(key: CENKey, maxTimestamp: Int64) -> [CEN] {
+
+        // Unclear why maxTimestamp is a parameter
+        let maxTimestamp = Date().coEpiTimestamp
```

Comment:
@i-schuetz 
Also not sure about the logic, but if we are to write unit tests for this function (and we definitely should), prviding maxTimestamp as input param is the way to go (parameter injection).

Reply:
This is meant mostly as a reminder to clarify the issue. I'd have deleted the parameter otherwise. I want to confirm that it's always the current timestamp.